common/system/tasks/trigger_manager.h

```diff
@@ -84,6 +94,9 @@ void triggerManager_triggerFire(uint8_t triggerNum, bool logFire);
 bool triggerManager_setTriggerConfig(uint8_t triggerNum,
                                      const char **configString);
 
+// returns the configuration of the trigger
+const TriggerState triggerManager_getExpectedTriggers(uint8_t i);
```

Comment:
This just returns one trigger so should probably be renamed to `triggerManager_getTriggerConnectivity` or something singular.